licenses/Apache-2.0.lre

```diff
@@ -151,26 +166,29 @@ https://opensource.org/licenses/Apache-2.0
       subsequently incorporated within the Work.
 
    (( 2. ))??
-   Grant of Copyright License. Subject to the terms and conditions of this
+   Grant of Copyright License. Subject to the terms and conditions of __1__
    License, each Contributor hereby grants to You a perpetual, worldwide,
    non-exclusive, no-charge, royalty-free, irrevocable copyright license to
    reproduce, prepare Derivative Works of, publicly display, publicly perform,
    sublicense, and distribute the Work and such Derivative Works in Source or
    Object form.
 
    (( 3. ))??
-   Grant of Patent License. Subject to the terms and conditions of this License,
+   Grant of Patent License. Subject to the terms and conditions of __1__ License,
    each Contributor hereby grants to You a perpetual, worldwide, non-exclusive,
-   no-charge, royalty-free, irrevocable (except as stated in this section)
+   no-charge, royalty-free, irrevocable (
+   (( except || expect ))
+   as stated in __1__ section)
    patent license to make, have made, use, offer to sell, sell, import, and
    otherwise transfer the Work, where such license applies only to those patent
    claims licensable by such Contributor that are necessarily infringed by their
    Contribution(s) alone or by combination of their Contribution(s) with the
    Work to which such Contribution(s) was submitted. If You institute patent
    litigation against any entity (including a cross-claim or counterclaim in a
-   lawsuit) alleging that the Work or a Contribution incorporated within the
+   ((lawsuit || luit))
```

Comment:
Was this an AWS remover? :-)

Reply:
Most likely! I found a bunch of search-and-replace artifacts, like "rebundleion" ("reproduction" with s/product/bundle/)